.gitlab-ci.yml

```diff
@@ -140,6 +140,44 @@ test_py39_mil_intel:
     WHEEL_PATH: build/dist/*cp39*10_15*
     REQUIREMENTS: reqs/test.pip
 
+test_py39_backends_intel:
+  <<: *test_macos_pkg
+  tags:
+    - macOS13
+  dependencies:
+    - build_wheel_macos_py39_intel
+  variables:
+    WHEEL_PATH: build/dist/*cp39*10_15*
+    TEST_PACKAGE: coremltools.converters.mil.backend
```

Comment:
Just curious: Why don't we test the whole `coremltools.converters.mil` package, so in the future we don't need to keep adding test entries?

Reply:
That's a good question. We certainly could do that and that would be more straightforward.

The reason is getting CI runs done quicker. In terms of the running time of unit tests, most of that is under `coremltools.converters.mil` since it contains all the TensorFlow tests, PyTorch tests and other tests.

Having them as separate GitLab jobs allows the TensorFlow tests, PyTorch tests and other tests to all run in parallel.